tools/src/test/python/dlpx/virtualization/_internal/test_plugin_importer.py

```diff
@@ -287,7 +287,7 @@ def test_import_error(mock_file_util, plugin_config_file,
     @staticmethod
     @pytest.mark.parametrize(
         'entry_point,expected_error',
-        [('bad_syntax:plugin', "SDK Error: invalid syntax")])
+        [('bad_syntax:plugin', "SDK Error: expected ':'")])
```

Comment:
nit: Is this change expected? Not sure if there are always syntax errors because of ignoring ":".

Reply:
Yes, it is just that with Python 3.11 the message got changed and became more precise.